code/modules/recycling/disposal/pipe.dm

```diff
@@ -17,6 +17,10 @@
 	var/initialize_dirs = NONE // bitflags of pipe directions added on init, see \code\_DEFINES\pipe_construction.dm
 	var/flip_type // If set, the pipe is flippable and becomes this type when flipped
 	var/obj/structure/disposalconstruct/stored
+	//SKYRAT EDIT: HURTSPOSALS
+	///whether a disposal pipe will hurt if a person changes direction
+	var/fluffy = FALSE
```

Comment:
Needs a meaningful name and the comment needs to make sense with the variable ;)
```suggestion
	/// Whether a disposal pipe will hurt if a person changes direction. `FALSE` for hurting, `TRUE` to prevent making them hurt.
	var/padded_corners = FALSE
```